packages/swap/src/utils/boost.ts

```diff
@@ -40,3 +41,11 @@ export const getBoostFeeBpsForAmount = async ({
     maxBoostFeeBps,
   };
 };
+
+export const getBoostChainflipBlocksDelayForChain = async (chain: Chain): Promise<number> => {
+  const boostDelay = await prisma.boostDelayChainflipBlocks.findFirst({
```

Comment:
we usually read these values from an RPC to ensure they are the latest, because the processor has a delay. here the impact is minimal if we have some stale data but still think an RPC would be preferable. seems like this value should be added to the ingress/egress environment RPC?

Reply:
However, i still see this being used on `main`